00-Install_and_Setup/environment.yml

```diff
@@ -31,4 +31,3 @@ dependencies:
     - "glue-vispy-viewers>=0.6"
     - "glueviz>=0.9.1"
     - astroquery
-    - --pre
```

Comment:
I am not sure about removing the `--pre`. People would be getting the last release, not the latest pre-release. Do we want this?

Reply:
I think this is what we don't want. There are a couple of workarounds I saw online. Or we could put a Dockerfile in the root directory for Binder to use.

@adrn does the Flatiron Binder use the repo2docker package? I installed it and it choked on this line.

Reply:
The ``--pre`` was there for astroquery only. I would say we need to keep that on for now.

Reply:
for the binder it'll be OK to pin the version number to be an exact match for a dev version from pypi, but in the docs given for participants ``--pre`` should stay until we come around and change to the new version number scheme